### PR TITLE
Added recipe for cement

### PR DESCRIPTION
**Pull request by pmlandwehr**
This is version `2.6.2` of `cement`, not the current `2.10.2`, because the old version is a dependency for `elex`. `setuptools` was listed as a runtime requirement by `conda skeleton`, but wasn't needed for the local imports so I've removed it. We'll see what happens...

### CODE REVIEW

File: recipes/cement/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - cement
+    - cement.core
+    - cement.ext
+    - cement.utils
+    - examples
+
+about:
+  home: http://builtoncement.com
+  license: BSD License
+  # license_file: No MANIFEST included with build
```
Comment (jakirkham) [inline recipes/cement/meta.yaml]: Could we please add a PR to provide this upstream?

File: recipes/cement/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - cement
+    - cement.core
+    - cement.ext
+    - cement.utils
+    - examples
+
+about:
+  home: http://builtoncement.com
+  license: BSD License
```
Comment (jakirkham) [inline recipes/cement/meta.yaml]: Please specify number of clauses and drop `License`.

File: recipes/cement/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/cement/meta.yaml]: Submitted an issue - see [issue 405](https://github.com/datafolklabs/cement/issues/405)

File: recipes/cement/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - cement
+    - cement.core
+    - cement.ext
+    - cement.utils
+    - examples
+
+about:
+  home: http://builtoncement.com
+  license: BSD 3-Clause
+  # license_file: No MANIFEST included with build
```
Comment (jakirkham) [inline recipes/cement/meta.yaml]: Please raise an issue upstream.

Comment (pmlandwehr) [inline recipes/cement/meta.yaml]: Done - see [issue 405](https://github.com/datafolklabs/cement/issues/405)

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**